libs/core/mappers/anime-parameters.mapper.ts

```diff
@@ -0,0 +1,49 @@
+import { AnimeParametersDto } from '../dtos/anime-parameters.dto';
+import { AnimeParameters } from '../models/anime-parameters';
+import { AnimeSortingField } from '../models/anime-sorting-field';
+import { SortingDirection, SortingParameters } from '../models/sorting-parameters';
+
+import { AnimeTypeMapper } from './anime-type.mapper';
+
+export namespace AnimeParametersMapper {
+
+	/**
+	 * Map model to dto for anime list parameters.
+	 * @param model Model for anime list parameters.
+	 */
+	export function toDto(model: AnimeParameters): AnimeParametersDto {
+		return {
+			limit: model.pageSize,
+			offset: model.pageSize * (model.pageNumber - 1),
+			ordering: mapSorting(model) ?? '',
+			type__in: model.animeTypes
+				.map(animeTypeModel => AnimeTypeMapper.toDto(animeTypeModel))
+				.toString(),
```

Comment:
In the JSDoc there is a comment:
> Types are separated by comma.

We can make our intentions more explicit by using `.join` instead of '.toString':
```suggestion
			type__in: model.animeTypes
				.map(animeTypeModel => AnimeTypeMapper.toDto(animeTypeModel))
				.join(','),
```